### DISCUSSION TITLE
Updating Fooocus??

### DISCUSSION THREAD

**Discussion by EddieSoloProjects** [lllyasviel/Fooocus#1335] [General]
This is a bit of a Noob Question. But does Fooocus update automatically or is their a .bat file I need to run?

**Comment by jepjoo** [comment]
Updates automatically when you start it.

**Reply by EddieSoloProjects** [reply]
Thanks :)

**Comment by Alireza2317** [comment]
Does anyone have an idea why i get this `Update failed.` at the output when i run the entry_with_update.py file?
is there another way to update fooocus?

**Reply by f0n51** [reply]
@Alireza2317 You should raise an issue and provide your full console log over there ;)
Nevertheless, depending on your operation system, you can easily do a full installation from scratch to get the latest version of Fooocus.

**Reply by mashb1t** [reply]
^  this and probably you have local changes, either in presets or other files. There is already a PR for presets.
